src/app/navigation/header/header.component.ts

```diff
@@ -16,30 +17,38 @@ export class HeaderComponent implements OnInit {
 
   ngOnInit() {
     this.datastorageService.isAuth.subscribe(
-      isloggedIn =>{
+      isloggedIn => {
         this.loggedInUser = isloggedIn;
       }
     )
```

Comment:
convert to a single line. use {} only when your subscribe logic is of more than 1 line.